### PR TITLE
Add getting started instructions for GTK4

### PR DESCRIPTION
**Pull request by viseztrance**
This ports the [C getting started instructions](https://docs.gtk.org/gtk4/getting_started.html), up to the building applications section.

### CODE REVIEW

**Review by viseztrance — COMMENTED** [review]
These should point to docs, but I can't find these anywhere on https://ruby-gnome.github.io/ruby-gnome/doc/dev/, so for now I'm pointing to their C apis.

File: gtk4/sample/examples/window_default.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+# example from https://gitlab.gnome.org/GNOME/gtk/blob/main/examples/window-default.c
```
Comment (kou) [inline gtk4/sample/examples/window_default.rb:19]: Could you also add license information for the original code?

```suggestion
# example from https://gitlab.gnome.org/GNOME/gtk/blob/main/examples/window-default.c
#
# License: LGPL2.1+
```

File: gtk4/sample/getting-started/README.md [hunk 2]
```diff
@@ -1,4 +1,357 @@
...
+  window.set_default_size(200, 200)
+  window.show
+end
+
+application.run
+```
+
+When creating a Gtk::Application you need to pick an application identifier (a name) and input to `Gtk::Application#new` as parameter. For this example *org.gtk.example* is used but for choosing an identifier for your application see this [guide](https://wiki.gnome.org/HowDoI/ChooseApplicationID).
+
+Lastly `Gtk::Application#new` takes a `Gio::ApplicationFlags` constant as input for your application, if your application would have special needs (those constants can be replaced by theirs respective symbol ie. `Gio::ApplicationFlags::FLAGS_NONE` == `:flags_none`). You must know that `GApplication` ignores arguments passed to `g_application_run()` on the Windows systems. It always uses command line arguments even when we pass an empty array to g_application_run().
```
Comment (kou) [inline gtk4/sample/getting-started/README.md:36]:
```suggestion
Lastly `Gtk::Application#new` takes a `Gio::ApplicationFlags` constant as input for your application, if your application would have special needs (those constants can be replaced by theirs respective symbol ie. `Gio::ApplicationFlags::FLAGS_NONE` == `:flags_none`). You must know that `Gio:::Application` ignores arguments passed to `Gio::Application#run` on the Windows systems. It always uses command line arguments even when we pass an empty array to `Gio::Application#run`.
```

File: gtk4/sample/examples/window_default.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env ruby
+#
+# Copyright (C) 2013-2022  Ruby-GNOME Project Team
```
Comment (kou) [inline gtk4/sample/examples/window_default.rb:3]:
```suggestion
# Copyright (C) 2022  Ruby-GNOME Project Team
```

**Review by kou — COMMENTED** [review]
Thanks!

**Comment by kou** [comment]
> These should point to docs, but I can't find these anywhere on https://ruby-gnome.github.io/ruby-gnome/doc/dev/, so for now I'm pointing to their C apis.

It makes sense.

**Pull request merged**